odxtools/diaglayer.py

```diff
@@ -476,29 +476,10 @@ def get_send_id(self):
                 # fragment of the ASAM MCD2-D standard.
                 return int(com_param.value[4])
 
-    def get_can_func_req_id(self):
```

Comment:
are the `get_can_func_req_id()`,  `get_doip_logical_gateway_address()` and `get_doip_logical_functional_address()` methods not required/useful?

Reply:
I added these functions, but never used it. All our tests are done with DoIP addresses. I generally keep the code lean and remove the parts that we don't use anymore. I didn't realize I was working on an open-source library and this could be used by someone else. My bad. Added them back.

Reply:
no worries. both options are fine with me (I never used odxtools with DoIP so far, so thanks for making my future life easier :)), I was just curious what motivated you.